packages/core/src/browser/menu/browser-menu-plugin.ts

```diff
@@ -31,6 +31,11 @@ import { ApplicationShell } from '../shell';
 import { CorePreferences } from '../core-preferences';
 import { PreferenceService } from '../preferences/preference-service';
 
+/**
+ * Enable/Disable the new focus adaptations
+ */
+const ENABLE_FOCUS_ADAPTATIONS = true;
```

Comment:
Not sure why we need this: the new behavior either correct or it's not.

Reply:
We don't need it. I marked that commit as wip as I'm not yet satisfied with the changes and I wanted to mark with what I did in the code. However I wanted to push so that others, like you, could take a look if necessary.

Before merging I will definitely remove that constant again.

Reply:
I removed it again